### DISCUSSION TITLE
installation of rtmpose on ubuntu 20.04

### DISCUSSION THREAD

**Discussion by sidh1603** [open-mmlab/mmpose#2577] [General]
Hi ,

Could you please send the steps to install rtmpose on ubuntu 20.04 , because i tried to install using github , but i got confused and after that it did not work.

Thanks

**Comment by Tau-J** [comment]
Just follow the [installation guide](https://mmpose.readthedocs.io/en/dev-1.x/installation.html) of mmpose

**Comment by sidh1603** [comment]
Hi,Thanks for your reply, i have done this step , when i have installed mmpose i got a folder by the name project , inside project there is a folder for rtmpose , there is got confused, so could you please explain me the steps to install rtmpose after installing mmpose.ThanksSent from my iPhoneOn Jul 27, 2023, at 3:06 AM, Tau ***@***.***> wrote:﻿
Just follow the installation guide of mmpose

**Comment by sidh1603** [comment]
Hi After installing mmpose , i tried to run this command as per your git
hub documentations

python demo/topdown_demo_with_mmdet.py
projects/rtmpose/rtmdet/person/rtmdet_nano_320-8xb32_coco-person.py
https://download.openmmlab.com/mmpose/v1/projects/rtmpose/rtmdet_nano_8xb32-100e_coco-obj365-person-05d8511e.pth

projects/rtmpose/rtmpose/body_2d_keypoint/rtmpose-m_8xb256-420e_coco-256x192.py

https://download.openmmlab.com/mmpose/v1/projects/rtmposev1/rtmpose-m_simcc-aic-coco_pt-aic-coco_420e-256x192-63eb25f7_20230126.pth
    --input {YOUR_TEST_IMG_or_VIDEO}     --show

i got this error File "demo/topdown_demo_with_mmdet.py", line 203, in main
    input_type = mimetypes.guess_type(args.input)[0].split('/')[0]
AttributeError: 'NoneType' object has no attribute 'split'


please tell the steps so that i am able to run rtmpose

THanks

**Reply by Tau-J** [reply]
Did you pull the latest code? It seems that you are using a previous version of demo script. I think you can clone and install again.

**Comment by sidh1603** [comment]
HI,,


Where i will find this

{PATH_TO_CHECKPOINT}/rtmdet_nano_8xb32-100e_coco-obj365-person-05d8511e.pth \
    projects/rtmpose/rtmpose/body_2d_keypoint/rtmpose-m_8xb256-420e_coco-256x192.py
\
    {PATH_TO_CHECKPOINT}/rtmpose-m_simcc-aic-coco_pt-aic-coco_420e-256x192-63eb25f7_20230126.pth

i am not able to get .pth

**Reply by Tau-J** [reply]
Hi，you can download the coresponding `.pth` file from [the readme](https://github.com/open-mmlab/mmpose/tree/main/projects/rtmpose)

**Comment by sidh1603** [comment]
Hi,i have clone dev 1.x versionSent from my iPhoneOn Jul 30, 2023, at 9:06 PM, Tau ***@***.***> wrote:﻿
Did you pull the latest code? It seems that you are using a previous version of demo script. I think you can clone and install again.

**Comment by sidh1603** [comment]
Hi,could you please send that which version i have to use.ThanksSiddharthSent from my iPhoneOn Jul 30, 2023, at 9:47 PM, Siddharth Rana ***@***.***> wrote:﻿Hi,i have clone dev 1.x versionSent from my iPhoneOn Jul 30, 2023, at 9:06 PM, Tau ***@***.***> wrote:﻿
Did you pull the latest code? It seems that you are using a previous version of demo script. I think you can clone and install again.
